packages/server/assets/core/typedefs/apitoken.graphql

```diff
@@ -55,6 +55,28 @@ input AppTokenCreateInput {
   limitResources: [TokenResourceIdentifierInput!]
 }
 
+"""
+A token used to enable an embedded viewer for a private project
+"""
+type EmbedToken {
+  tokenId: String!
+  projectId: String!
+  userId: String!
+  modelIds: String!
+  createdAt: DateTime!
+  lifespan: BigInt!
+  lastUsed: DateTime!
+}
+
+input EmbedTokenCreateInput {
+  projectId: String!
+  """
+  The model(s) and version(s) string used in the embed url
+  """
+  modelIds: String!
```

Comment:
Is this the resourceIdString? The one formatted like 'modelId,modelId@versionId,objectId'. If it is, we should use this terminology everywhere and we even have utils for serializing & deserializing

Reply:
yes, didn't realize we had a name for it. Fixed all references